### ISSUE TITLE
Windoes 7

### ISSUE THREAD

**Issue by harryray33** [winsiderss/systeminformer#2122]
Do any of the versions work with Win 7?
Thanks.

**Comment by MagicAndre1981** [comment]
The application works fine in Windows 7:

![image](https://github.com/winsiderss/systeminformer/assets/8274816/c83a3693-75cb-4b6e-8790-594d54654f72)

Only the driver can't be loaded as the driver needs at least Windows 10

![image](https://github.com/winsiderss/systeminformer/assets/8274816/30047757-9314-4dd8-820e-2cef3dbe6ab3)

**Comment by i486** [comment]
Could someone please share the absolute last version of SystemInformer that supports loading kernel driver on Windows 7? I recall someone previously mentioning this version somewhere, but I can't locate it.

**Comment by jxy-s** [comment]
System Informer kernel driver does not support Windows 7. System Informer supports Windows 7 but only without the kernel driver. If you want kernel functionality for Windows 7 you must use the legacy project Process Hacker: https://processhacker.sourceforge.io/downloads.php

**Comment by i486** [comment]
> System Informer kernel driver does not support Windows 7. System Informer supports Windows 7 but only without the kernel driver. If you want kernel functionality for Windows 7 you must use the legacy project Process Hacker: https://processhacker.sourceforge.io/downloads.php

If there has never been a System Informer build that supported loading the kernel driver on Windows 7, then I was probably thinking of Process Hacker (but not the old v2.39 release). If I remember correctly, there was a transitional period when Process Hacker was still being developed before its rebranding to System Informer. It was likely a Process Hacker 3.x alpha or snapshot build, and the kernel driver was working at the time. Unfortunately, I lost that version with the drive.

Basically, I need the latest public beta, alpha, or snapshot build from the past that supported loading the driver.

**Comment by jxy-s** [comment]
All the builds are located here: https://github.com/winsiderss/si-builds/releases

You're welcome to try to go use the oldest one, but I assure you it won't work.

**Comment by MagicAndre1981** [comment]
that are the builds after rename and driver rework that removed Win7 support. Last PH build I found on my disks that loads driver is 3.0.4600 from 3 years ago

[processhacker-3.0.4600-bin.zip](https://github.com/user-attachments/files/18722476/processhacker-3.0.4600-bin.zip)

I have no idea if someone else has a newer build

**Comment by i486** [comment]
> [processhacker-3.0.4600-bin.zip](https://github.com/user-attachments/files/18722476/processhacker-3.0.4600-bin.zip)
> 
> I have no idea if someone else has a newer build

Yeah, that’s exactly what I was talking about. Thanks for sharing!

Also, I just found a newer build from an old VM backup!

[processhacker-3.0.4953-final.zip](https://github.com/user-attachments/files/18722521/processhacker-3.0.4953-final.zip)

If anyone has an even newer version, I’d love to get my hands on it!

**Comment by i486** [comment]
> that are the builds after rename and driver rework that removed Win7 support. Last PH build I found on my disks that loads driver is 3.0.4600 from 3 years ago
> 
> [processhacker-3.0.4600-bin.zip](https://github.com/user-attachments/files/18722476/processhacker-3.0.4600-bin.zip)
> 
> I have no idea if someone else has a newer build

Yeah, that’s exactly what I was talking about. Thanks for sharing!

Also, I just found a newer build from an old VM backup!

If anyone has an even newer version, I’d love to get my hands on it!

[processhacker-3.0.4953-final.zip](https://github.com/user-attachments/files/18722544/processhacker-3.0.4953-final.zip)
